.gitignore

```diff
@@ -1,2 +1,5 @@
 bin/
 *.log
+*.json
```

Comment:
Please use directory prefixes. It's possible for a codebase to contain `.json` files, so I'd prefer just to ignore files in a directory made for persistent storage.